app/code/Magento/InventoryAdminUi/Test/Mftf/Test/ReorderButtonConfigurableProductDefaultStock.xml

```diff
@@ -0,0 +1,117 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
+    <test name="ReorderButtonConfigurableProductDefaultStock">
+        <annotations>
+            <stories value="Configurable Product Default Stock."/>
+            <title value="Reorder Button Visibility with Configurable Product on Default Stock."/>
+            <description value="Verify,reorder button is not visible after shipment created for order with configurable product on default source."/>
+            <testCaseId value="https://app.hiptest.com/projects/69435/test-plan/folders/735208/scenarios/2630678"/>
```

Comment:
* ReorderButtonConfigurableProductTestStock
* StorefrontButtonReorderForTheOrderWithConfigurableProductOnDefaultStockInCustomerCabinetOnHomepageIsNotPresentIfProductSoldOutTest

Reply:
Removed

Reply:
Will be removed in next PR